docs/semgrep-ci/manual-ci-setup.md

```diff
@@ -0,0 +1,414 @@
+
+By refining a job's parameters, you are able to achieve the following goals:
+
+* **Run Semgrep on a schedule.** Run full scans on mainline branches at a least intrusive time on developer teams.
+* **Run Semgrep with custom rules.** Apply rules specific to your organization's business goals and coding conventions.
+* **Run Semgrep when an event triggers.** Run Semgrep when a pull or merge request (PR or MR) is created. These event triggers or event hooks are dependent on your CI provider. 
+* **Run Semgrep on relevant files and blocks of code.** Configure Semgrep to ignore files and folders such as test files, configuration files, and files from other vendors.
+* **Pass a Semgrep CI job even when scans report findings.** By default, manual configurations fail when any finding is detected. Configure Semgrep to pass CI jobs even when findings are reported (a **fail open** state).
+
+## Limitations of manually configured Semgrep CI scans 
+
+* Findings are not tiered based on severity and actions cannot be undertaken based on severity. This means that either any finding will cause the job to fail, or any finding still allows the job to pass.
+* Findings are dumped to a log and are not tracked over time, so there is no record of a finding's state, such as opened, closed, or ignored.
+
+## Setting up the CI job
+
+### Introduction
+
+ `semgrep ci` is the command used to run Semgrep in a CI environment. In most cases, this is the recommended command to run in the CI job. It is a subset of the `semgrep scan` command. Features of this command include:
+ 
+ * `semgrep ci` is git-aware. This means it is able to detect branches and git states. 
+ * `semgrep ci` makes use of environment variables to configure its behavior. 
+ * `semgrep ci` performs a diff-aware scan by default. This means it only scans changes in files when run on a pull or merge request.
+ * `semgrep ci` can be run on a local `.git` repository at any time to test its behavior before running it within a CI environment.
+
+:::note
+An alternative method of running Semgrep in your CI is to run the `semgrep scan` command, passing rules to scan, for example `semgrep scan --config auto` which performs a full scan of the repository every time the job is run.
+:::
+
+### GitHub Actions
+
+To add Semgrep into your GitHub Actions pipeline, do the following steps.
+
+1. Create a `semgrep.yaml` configuration file to add a Semgrep command as part of your pipeline. Refer to the [GitHub Actions code snippet](#github-actions-code-snippet).
+2. Optional: You can also review the [GitHub Actions workflow syntax](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions) to help with editing the configuration file.
+3. Commit the configuration file into the `./github/workflows` folder within the target repository. The Semgrep job starts automatically upon detecting the `semgrep.yaml` commit in the previous step
```

Comment:
When I tested it on GHA, the committing of the job itself starts the scan. Maybe because of push events? Hi @holdenmcg, do you know if the job starts as soon as the file is committed _for all providers?_ I may have just assumed this for GHA.

Reply:
Regarding the committing starting the job; if the file is configured to run on 'push' events, then it will trigger the job when you commit the file. 
Some providers do this automatically regardless of whether the push event is present (since you're changing the CI config, so it automatically knows to re-run the job) but this isn't always the case.

Reply:
(Thanks Holden! Sorry for the late acknowledgement. I'm updating the steps.)